Makefile

```diff
@@ -0,0 +1,42 @@
+FMT_OPTS = -co -XTypeApplications -o -XImportQualifiedPost
+FIND_OPTS = src test -type f -name '*.hs'
+
+build:
+	cabal build
+
+test:
+	cabal test
+
+# Dogfood the dev version
+analyze:
+	cabal run fossa -- analyze --output --debug --filter 'cabal@./'
+
+# Copy the built binary into the local root
+fossa:
+	cp $(shell cabal list-bin fossa) ./fossa
+
```

Comment:
I'd just call this `make install`, not `make fossa` or `make install-local`.

Reply:
It's intentionally given the same name as the file it produces.  `make` will not rebuild it if it already exists, which saves some time.